devservices/config.yml

```diff
@@ -90,6 +111,330 @@ services:
+          --enforce-schema
+        ]
+    environment:
+      PYTHONUNBUFFERED: 1
+      SNUBA_SETTINGS: docker
+      DEBUG: 1
+      CLICKHOUSE_HOST: clickhouse
+      CLICKHOUSE_PORT: 9000
+      CLICKHOUSE_HTTP_PORT: 8123
+      DEFAULT_BROKERS: kafka:9093
+      REDIS_HOST: redis
+      REDIS_PORT: 6379
+      REDIS_DB: 1
+    platform: linux/amd64
+    extra_hosts:
+      host.docker.internal: host-gateway
+    networks:
+      - devservices
+    labels:
+      - orchestrator=devservices
+    restart: unless-stopped
+  profile-chunks-consumer:
+    image: ghcr.io/getsentry/snuba:latest
+    healthcheck:
+      test: curl -f http://localhost:1218/health_envoy
+      interval: 5s
+      timeout: 5s
+      retries: 3
+    command: [
+          snuba,
+          rust-consumer,
+          --storage=profile_chunks,
+          --consumer-group=profile_chunks_group,
+          --use-rust-processor,
+          --auto-offset-reset=latest,
+          --no-strict-offset-reset,
+          --enforce-schema
+        ]
+    environment:
+      PYTHONUNBUFFERED: 1
+      SNUBA_SETTINGS: docker
+      DEBUG: 1
+      CLICKHOUSE_HOST: clickhouse
+      CLICKHOUSE_PORT: 9000
+      CLICKHOUSE_HTTP_PORT: 8123
+      DEFAULT_BROKERS: kafka:9093
+      REDIS_HOST: redis
+      REDIS_PORT: 6379
+      REDIS_DB: 1
+    platform: linux/amd64
+    extra_hosts:
+      host.docker.internal: host-gateway
+    networks:
+      - devservices
+    labels:
+      - orchestrator=devservices
+    restart: unless-stopped
+  functions-consumer:
+    image: ghcr.io/getsentry/snuba:latest
+    healthcheck:
+      test: curl -f http://localhost:1218/health_envoy
+      interval: 5s
+      timeout: 5s
+      retries: 3
+    command: [
+          snuba,
+          rust-consumer,
+          --storage=functions_raw,
+          --consumer-group=functions_group,
+          --use-rust-processor,
+          --auto-offset-reset=latest,
+          --no-strict-offset-reset,
+          --enforce-schema
+        ]
+    environment:
+      PYTHONUNBUFFERED: 1
+      SNUBA_SETTINGS: docker
+      DEBUG: 1
+      CLICKHOUSE_HOST: clickhouse
+      CLICKHOUSE_PORT: 9000
+      CLICKHOUSE_HTTP_PORT: 8123
+      DEFAULT_BROKERS: kafka:9093
+      REDIS_HOST: redis
+      REDIS_PORT: 6379
+      REDIS_DB: 1
+    platform: linux/amd64
+    extra_hosts:
+      host.docker.internal: host-gateway
+    networks:
+      - devservices
+    labels:
+      - orchestrator=devservices
+    restart: unless-stopped
+  uptime-consumer:
+    image: ghcr.io/getsentry/snuba:latest
+    healthcheck:
+      test: curl -f http://localhost:1218/health_envoy
+      interval: 5s
+      timeout: 5s
+      retries: 3
+    command: [
+          snuba,
+          rust-consumer,
+          --storage=uptime_monitor_checks,
+          --consumer-group=snuba-uptime-results,
+          --use-rust-processor,
+          --auto-offset-reset=latest,
+          --no-strict-offset-reset,
+          --enforce-schema
+        ]
```

Comment:
These consumers have a lot of duplication between them, it would be a good idea to create an issue to look into enabling templating in these configs down the road.